src/main/java/core/basesyntax/service/impl/operation/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.service.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
The class `BalanceOperation` should not start with an empty line. Please remove the empty line to comply with the checklist.

Reply:
--